src/rulesets/Base/fastmath_able.jl

```diff
@@ -230,9 +260,10 @@ let
     non_transformed_definitions = intersect(fastable_ast.args, fast_ast.args)
     filter!(expr->!(expr isa LineNumberNode), non_transformed_definitions)
     if !isempty(non_transformed_definitions)
-        error(
-            "Non-FastMath compatible rules defined in fastmath_able.jl. \n Definitions:\n" *
-            join(non_transformed_definitions, "\n")
+        @warn(
+            "Non-FastMath compatible rules defined in fastmath_able.jl.", # \n Definitions:\n" *
+            # join(non_transformed_definitions, "\n")
+            non_transformed_definitions
```

Comment:
Was this intentional?

Reply:
This is because the error doesn't play well with Revise. I can revert it when done fidding. Or make it an `@error` to save the next person the hassle.

Reply:
Hmm, odd... Would be good to at least have a comment about that.

Reply:
Now restored, with a comment.